crates/daemon/src/wallet.rs

```diff
@@ -512,10 +579,42 @@ mod tests {
                     time_to_lock,
                 },
                 blockchain_client: (),
+                db: None,
             })
         }
     }
 
+    impl Actor<ElectrumBlockchain, Tree> {
+        pub async fn spawn_test(
+            data_dir: PathBuf,
+        ) -> Result<(xtra::Address<Self>, watch::Receiver<Option<WalletInfo>>)> {
+            // generate empty export seed
+            let mut export_seed = [0u8; 256];
+            thread_rng().fill(&mut export_seed);
```

Comment:
Given this test will run now regularly we should check if this seed indeed does not hold any Bitcoin 🤑

Reply:
@bonomat what do you have in mind exactly?